transcoder/docs/playback.md

```diff
@@ -0,0 +1,178 @@
+# Distributed transcoding and how playback works
+
+Kyoo provides videos via [HTTP live streaming](https://www.cloudflare.com/learning/video/what-is-http-live-streaming/) (HLS). HLS is comprised of two components: a "playlist" (using the `.m3u8` file extension), and "transport stream segments", or "segments" (using the `.ts` file extension). Playlists contain a set of segments, which are pieces of the video being streamed. When playing a video, the client first requests a playlist of the video, and then requests segments of the video, as needed.
```

Comment:
Segments aren't necessarily `.ts` files, some versions of HLS support fmp4 (see #542) and one day another segment format could be adopted.